tdrs-frontend/nginx/mime.types

```diff
@@ -0,0 +1,78 @@
+types {
+  text/html html htm shtml;
```

Comment:
nginx build pack required a mime.types file, this was a pretty inclusive one I found looking through the build packs examples.

Reply:
@riatzukiza Can you say a little bit about what the mime-type file does? I see some executable types in here and want to make sure all of these are needed. Thanks.

Reply:
The format is `mime/type` followed by a space delimited list of valid extensions. The file here was taken from the nginx buildpack samples.

Reply:
eg `  text/html html htm shtml;` tells the server to add `text/html` as the `mimetype` http response header for all requests that have either `html`, `htm`, or `shtml` extensions as a part of their path.